src/com/minseung/datastructure/LinkedListNode.java

```diff
@@ -225,37 +235,38 @@ public static void main(String[] args) {
         ll.append(3);
```

Comment:
I would create a helper method in this class which takes array of integer and returns linked list from the list. How do you think?